automated install/basic-install.sh

```diff
@@ -1770,18 +1770,49 @@ create_pihole_user() {
     printf "  %b %s..." "${INFO}" "${str}"
     # If the user pihole exists,
     if id -u pihole &> /dev/null; then
-        # just show a success
-        printf "%b  %b %s\\n" "${OVER}" "${TICK}" "${str}"
+        # if group exists
+        if getent group pihole; then
```

Comment:
Please redirect output of this check to /dev/null, else output will be printed to console:
```
root@dietpi:~# if getent group www-data; then :; fi
www-data:x:33:
```
=> `if getent group pihole > /dev/null 2>&1; then`